### PR TITLE
Firstrealcommit

### PR DESCRIPTION
**Pull request by colincoleman**

### CODE REVIEW

File: docker/sonarqube-aws-env/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM teliaoss/docker-sonarqube:7.3
```
Comment (itsdalmo) [inline docker/sonarqube-aws-env/Dockerfile]: I'm assuming the above is a temporary thing until they release their own version? Perhaps you could submit a PR upstream to add the images?

File: docker/sonarqube-aws-env/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+chown -R sonarqube:sonarqube $SONARQUBE_HOME
+
+if [ -z "$AWS_REGION" ]; then
```
Comment (itsdalmo) [inline docker/sonarqube-aws-env/entrypoint.sh]: Some concern about this listed here: https://github.com/telia-oss/terraform-aws-grafana/pull/8#discussion_r209541364

File: outputs.tf [hunk 3]
```diff
@@ -2,3 +2,10 @@
 # Output
 # ------------------------------------------------------------------------------
 
+output "target_group_arn" {
+  value = "${module.sonarqube-service.target_group_arn}"
+}
+
+output "sonarqube_URL" {
```
Comment (itsdalmo) [inline outputs.tf]: Think this can just be lowercase `url`.

File: policies.tf [hunk 4]
```diff
@@ -0,0 +1,79 @@
+resource "aws_iam_policy" "sonarqube-task-pol" {
+  name = "${var.name_prefix}-task-pol"
```
Comment (itsdalmo) [inline policies.tf]: task policy? Prefer if its typed out fully 😄

File: docker/sonarqube-aws-env/Dockerfile (hunk 1 above)
Comment (colincoleman) [reply docker/sonarqube-aws-env/Dockerfile]: Sonarqube and (temporarily?) stopped providing docker files and official docker images on Dockerhub.  Forked their git repo (GPL3....) and added files for the new versions.  Hopefully just temporary.

Comment (itsdalmo) [reply docker/sonarqube-aws-env/Dockerfile]: Roger!

**Pull request merged**